main.py

```diff
@@ -86,19 +90,31 @@ def extract_transform_and_load_covid_data(post_to_google_sheets=True):
     #     credentials=credentials,
     # )
 
-    covidtracking_df = extract_covidtracking_historical_data()
-    cdc_ili_df = extract_cdc_ili_data()
+    covidatlas_df = extract_covid_atlas_data()
+    # covidtracking_df = extract_covidtracking_historical_data()
```

Comment:
Are these intentinoally commented out, or was that just to speed up local development of your new feature?

Reply:
just for local dev!